core/domain/state_domain.py

```diff
@@ -2112,6 +2112,19 @@ def convert_html_fields_in_state(cls, state_dict, conversion_fn):
                             answer_group_index]['rule_specs'][
                                 rule_spec_index]['inputs']['x'][x_index] = (
                                     conversion_fn(x))
+
+        if 'written_translations' in state_dict.keys():
```

Comment:
Why `if`? We are expecting every state dict to have this field, right? (I mean this is a property of state object)

Reply:
Not all dicts in the tests for migrations have written_transalations as a key, that is why added this check. Only the later schemas have this as a key.